joint_limits_interface/package.xml

```diff
@@ -0,0 +1,39 @@
+<package format="3">
+  <name>joint_limits_interface</name>
+  <version>0.16.0</version>
+  <description>Interface for enforcing joint limits.</description>
+
+  <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
+  <maintainer email="enrique.fernandez.perdomo@gmail.com">Enrique Fernandez</maintainer>
+  <maintainer email="mathias.luedtke@ipa.fraunhofer.de">Mathias Lüdtke</maintainer>
+
+  <license>BSD</license>
+
+  <url type="website">https://github.com/ros-controls/ros_control/wiki</url>
```

Comment:
```suggestion
  <url type="website">https://github.com/ros-controls/ros2_control/wiki</url>
```

Reply:
FYI There is no ros2_controls Wiki yet, it just goes to the main code/README page